tests/Dfe.PlanTech.Web.E2ETests/cypress/e2e/pages/interstitial-page.cy.js

```diff
@@ -1,4 +1,5 @@
 describe("Interstitial page", () => {
+  // eslint-disable-next-line no-unused-vars
   let url;
```

Comment:
this looks like a valid error, do we need url? it seems like it gets assigned below but not used at all

Reply:
Actually @katie-gardner I think this needed - isn't it used by the login command?